pkg/controller/podgroup_test.go

```diff
@@ -154,7 +163,7 @@ func Test_Run(t *testing.T) {
 				return true, nil
 			})
 			if err != nil {
-
+				t.Fatal("Unexpected error", err)
```

Comment:
Thanks for the catch. A bit surprised nobody spotted this before...

`ErrorS` should suffice:

```suggestion
				t.ErrorS(err, "Unexpected phase for podGroup")
```

Let's follow the structured logging format - use t.

Reply:
Hi, t have no ErrorS method, it provide Fatal or Errorf or Error, do you mean klog.ErrorS?
`t.ErrorS undefined (type *"testing".T has no field or method ErrorS)`
I still think `t.Fatal` is better :)

Reply:
I misread... Apology for that.